GVRf/Framework/framework/src/main/java/org/gearvrf/GVRRenderTexture.java

```diff
@@ -48,9 +48,9 @@ public GVRRenderTexture(GVRContext gvrContext, int width, int height) {
      *            MSAA sample count.
      */
     public GVRRenderTexture(GVRContext gvrContext, int width, int height,
-            int sampleCount) {
+            int sampleCount, int number_views) {
```

Comment:
Please update the javadoc. Also please add a new ctor that uses 1 view by default. Like:
```
    public GVRRenderTexture(GVRContext gvrContext, int width, int height) {
        this(gvrContext, width, height, 1);
    }
```

Seems more sensible than making code changes to the demos.